omnibus/files/private-chef-cookbooks/private-chef/providers/pg_database.rb

```diff
@@ -10,33 +10,20 @@ def whyrun_supported?
 use_inline_resources
 
 action :create do
-  execute "create_database_#{new_resource.database}" do
-    command createdb_command
-    user node['private_chef']['postgresql']['username']
-    not_if {database_exist?}
-    retries 30
+  ruby_block "Maybe create database #{new_resource.database}" do
```

Comment:
Since this is already provider code, I don't see why we need to put this in a ruby_block.